File: src/main/java/bot/util/database/SqlScripts.java
package bot.util.database;

public class SqlScripts {
    public static final String SELECT_LANGUAGE = "SELECT language FROM settings where id = ?;";

    public static final String INSERT_LANGUAGE = "INSERT INTO settings VALUES (?, ?::\"lang\");";

    public static final String UPDATE_LANGUAGE = "UPDATE settings SET language=?::\"lang\" WHERE id=?;";
}

File: src/main/java/bot/util/database/SqlScripts.kt
package bot.util.database

object SqlScripts {
    const val SELECT_LANGUAGE = "SELECT language FROM settings where id = ?;"

    const val INSERT_LANGUAGE = "INSERT INTO settings VALUES (?, ?::\"lang\");"

    const val UPDATE_LANGUAGE = "UPDATE settings SET language=?::\"lang\" WHERE id=?;"
}